data/template/buf.connect-go.gen.yaml

```diff
@@ -0,0 +1,11 @@
+# This template is to generate connect go.
```

Comment:
Why did we need a separate file for this? This seems incorrect

Reply:
Should I move this to `data/template/buf.go-client.gen.yaml`?